### PR TITLE
Add supports for list-of-table options

### PR DESCRIPTION
**Pull request by sbabcoc**
Currently our json based configurations does not
support List of Maps (this is supported on Toml
format)

Issue details are available in
https://groups.google.com/g/selenium-developers/c/_UYMd3ZjQ0M

**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description
More details on this issue is available in https://groups.google.com/g/selenium-developers/c/_UYMd3ZjQ0M

### Motivation and Context
To support list of maps when dealing with json config files for Grid.

### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [X] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: .bazelignore [hunk 1]
```diff
@@ -3,3 +3,9 @@ java/client/build
 java/server/build
 node_modules
 java/build/production
+.ijwb
+bazel-bin
+bazel-genfiles
+bazel-out
+bazel-selenium
+bazel-testlogs
```
Comment (diemol) [inline .bazelignore]: Why is this needed? Doesn't bazel need to know this information?

Comment (sbabcoc) [reply .bazelignore]: * `.ijwb` is a folder created and used by IntelliJ IDEA
* The other folders are output folders created by Bazel during the build/test process. These are output folders that don't contain core project artifacts.
Without these "ignore" entries, Bazel and IntelliJ end up scanning these folders for project specifications and other build artifacts that they will never contain, resulting in warning messages and extending the time required to build the project for no purpose.

Comment (titusfortner) [reply .bazelignore]: the `.bazelignore` file is to specify:
> directories within the workspace that you want Bazel to ignore, such as related projects that use other build systems

I don't think these qualify as that.

As for IntelliJ, it isn't going to read the `.bazelignore` file to know what to index. You need to use its own "Mark directory as..." assignments and set to `Excluded`.

Comment (sbabcoc) [reply .bazelignore]: I reverted these changes, as they are tangential to the focus of this PR. The warning messages that are emitted when Bazel scans these folders don't affect the actual build output; these messages are just noise.

**Review by diemol — APPROVED** [review]
Thank you!

(Test failure is unrelated to this change).
